tree/ntuple/v7/doc/specifications.md

```diff
@@ -270,7 +270,6 @@ The flags field can have one of the following bits set
 | Bit      | Meaning                                                                    |
 |----------|----------------------------------------------------------------------------|
 | 0x01     | Repetitive field, i.e. for every entry $n$ copies of the field are stored  |
-| 0x02     | Alias field, the columns referring to this field are alias columns         |
```

Comment:
We could use it as a consistency check but I think we should go the other way round: try to avoid redundant information in the file format.  In this case, it also added non-negligible overhead to the serialization code (and would have been never used in the deserialization)